.jenkins/Dockerfile

```diff
@@ -33,14 +33,14 @@ RUN curl "https://get.docker.com/builds/`uname -s`/`uname -m`/docker-latest.tgz"
     mv docker/* /usr/bin &&\
     rm -rf docke*
 
-# Install Java 11 (default)
+# Install Java 11
```

Comment:
are old versions still needed or are we safe to remove alternative java version?